setup.py

```diff
@@ -151,7 +153,12 @@ def package_files(prefix: str, directory: str, extension: str):
     "wurlitzer>=3.0.2,<4",
 ]
 
-extra_deps["all"] = set(dep for deps in extra_deps.values() for dep in deps)
+extra_deps['mlperf'] = [
+    # TODO: switch to pip package when available: https://github.com/mlcommons/logging/issues/218
+    'git+https://github.com/mlperf/logging.git',
```

Comment:
(Creating a comment to track this TODO -- cannot merge this PR until this is resolved, as this will break pip builds.)